releasenotes/notes/0.45/removed_deprecation_0.20-17b0ea6a3980eb4f.yaml

```diff
@@ -1,18 +1,17 @@
 ---
-upgrade:
+upgrade_visualization:
   - |
-    The class constructor arguments `qregs`, `cregs`, 
-    `layout` and `global_phase` for :class:`.visualization.QCircuitImage`
+    The class constructor arguments ``qregs``, ``cregs``,
+    ``layout`` and ``global_phase`` for :class:`.visualization.QCircuitImage`
```

Comment:
Is this a public interface? If not we probably don't need a release note for it

Reply:
I highly doubt anyone has ever used it, but it is a public interface.